bookdown/03-optimization.Rmd

```diff
@@ -1,23 +1,28 @@
 # Model Optimization {#model-optim}
 
-[**Model Tuning**](#tuning)
+**Model Tuning**
 
 Machine learning algorithms have default values set for their hyperparameters.
 Irrespective, these hyperparameters need to be changed by the user to achieve optimal performance on the given dataset.
 A manual selection of hyperparameter values is not recommended as this approach rarely leads to an optimal performance.
 To substantiate the validity of the selected hyperparatmeters (= [tuning](#tuning)), data-driven optimization is recommended.
 In order to tune a machine learning algorithm, one has to specify (1) the [search space](#tuning-optimization), (2) the [optimization algorithm](#tuning-optimization) (aka tuning method) and (3) an evaluation method, i.e., a resampling strategy and a performance measure.
 
-In summary, the sub-chapter on [tuning](#tuning) illustrates how to: 
+A potential tuning process is illustrated in the graphic below: 
+
+```{r 03-optimization-001, echo = FALSE}
+knitr::include_graphics("images/tuning-rescaled.svg")
+```
 
+In summary, the sub-chapter on [tuning](#tuning) illustrates how to: 
 * undertake empirically sound [hyperparameter selection](#tuning)
 * select the [optimizing algorithm](#tuning-optimization) 
 * [trigger](#tuning-triggering) the tuning
 * [automate](#autotuner) tuning
 
-This [sub-chapter](#tuning) requires the package `mlr3-tuning`, an extension package which supports hyperparameter tuning.
+The sub-chapter on model tuning requires the package `mlr3-tuning`, an extension package which supports hyperparameter tuning.
```

Comment:
pkg name (still) misspelled, also please use the `r mlr_pkg()` way to link to a pkg.